ember-file-upload/addon/components/file-upload.hbs

```diff
@@ -4,6 +4,7 @@
   ...attributes
   {{this.bindListeners}}
 >
+  {{~!~}}
```

Comment:
Can you explain or reference what this is doing (beyond a forum post).

Not sure I'm comfortable adding this if it's not a documented feature of glimmer as it may break in future versions.

Reply:
My understanding is it's coming from handlebars, not Glimmer. It looks to be combining the syntax for a comment with the syntax for white-space control: https://handlebarsjs.com/guide/expressions.html#whitespace-control.

Alternatively you could remove the white-space in this template file manually, but I was afraid that would be easier to regress.

Reply:
Ok fair enough. In the interest of easing migration I'll merge this. As I mentioned in the issue, though, this component is about to be deprecated anyway.

Reply:
Understood on the deprecation, but thanks for merging a stop-gap -- much appreciated!